### PR TITLE
Mv tx history cnt to db table

### PR DESCRIPTION
**Pull request by egaxhaj-figure**
## Description

Added Materialized views for `DAY` and `HOUR` to help with Tx History Graph with initial load time on the `14 Day Transaction History` graph on the [Dashboard](https://explorer.provenance.io/dashboard)

The views get refreshed when a block is written to the `block_cache` table. See:
* [Blocks.kt](https://github.com/provenance-io/explorer-service/blob/265aefaeee313f6b43eabfd728aaaeed812008c0/service/src/main/kotlin/io/provenance/explorer/domain/entities/Blocks.kt#L65:#L101)
* [V1_21__Add_tx_history_materialized_views.sql](https://github.com/provenance-io/explorer-service/blob/265aefaeee313f6b43eabfd728aaaeed812008c0/database/src/main/resources/db/migration/V1_21__Add_tx_history_materialized_views.sql)

closes: #117 

---

Before we can merge this PR, please make sure that all the following items have been
checked off. If any of the checklist items are not applicable, please leave them but
write a little note why.

- [x] Targeted PR against correct branch (see [CONTRIBUTING.md](https://github.com/provenance-io/explorer/blob/main/CONTRIBUTING.md#pr-targeting))
- [x] Linked to Github issue with discussion and accepted design OR link to spec that describes this work.
- [ ] Wrote unit and integration
- [x] Added a relevant changelog entry to the `Unreleased` section in `CHANGELOG.md`
- [x] Re-reviewed `Files changed` in the Github PR explorer
- [ ] Review `Codecov Report` in the comment section below once CI passes

### CODE REVIEW

File: service/src/main/kotlin/io/provenance/explorer/domain/entities/Blocks.kt [hunk 1]
```diff
@@ -156,10 +179,12 @@ class BlockProposerRecord(id: EntityID<Int>) : IntEntity(id) {
...
-            BlockProposerRecord.findById(height) ?: new(height) {
-                this.proposerOperatorAddress = proposer!!
-                this.blockTimestamp = timestamp!!
-            }.apply {
+            (
+                BlockProposerRecord.findById(height) ?: new(height) {
+                    this.proposerOperatorAddress = proposer!!
+                    this.blockTimestamp = timestamp!!
+                }
+            ).apply {
```
Comment (fkneeland-figure) [inline service/src/main/kotlin/io/provenance/explorer/domain/entities/Blocks.kt:187]: Did you intend to change this back from when I removed the parenthesis to make the linter not so ugly?

Comment (crussell-figure) [reply service/src/main/kotlin/io/provenance/explorer/domain/entities/Blocks.kt:187]: The parens are necessary. They should be put back

**Comment by crussell-figure** [comment]
@egaxhaj-figure looks like klint failed. There are notes on linting in the README.md

**Review by crussell-figure — APPROVED** [review]
LGTM

**Pull request merged**